linuxNew/README.md

```diff
@@ -1,10 +1,80 @@
-# Linux Packages of Eclipse Adoptium 
+# Linux Packages of Eclipse Adoptium
 
-## RPMs for Fedora/Red Hat
+We package for Debian, Red Hat, Suse (e.g. Yum and RPM based) Linux distributions.
```

Comment:
Let's not mix the package installation manager with the name of the packages. Also capitalise SUSE correctly:
```suggestion
We package for Debian, Red Hat, SUSE (e.g. DEB and RPM based) Linux distributions.
```